src/main/g8/build.sbt

```diff
@@ -3,23 +3,22 @@ lazy val root = (project in file(".")).
   settings(
     inThisBuild(List(
       organization := "$organization$",
-      scalaVersion := "2.11.8"
+      scalaVersion := "2.11.11"
```

Comment:
Oh, good point, sounds better to keep 2.11.8 then. I rolled back that change in the commit below.